packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/useHandleUpload.ts

```diff
@@ -95,17 +110,25 @@ const mergeSelectedTasks = (
 export function useHandleUpload({
   prefix,
   preventOverwrite,
+  batchSize = DEFAULT_BATCH_SIZE,
 }: {
   prefix: string;
   preventOverwrite: boolean;
+  batchSize?: number;
 }): [
   tasks: CancelableTask[],
   handleUpload: () => void,
   handleFileSelect: (files: File[]) => void,
+  handleCancel: () => void,
```

Comment:
This cancels all right?

Reply:
yes, `handleCancel` will cancel all tasks that are able to be canceled

Reply:
`handleCancelAll`?

Reply:
sounds good, will make that change